packages/libs/package.ts

```diff
@@ -34,7 +34,6 @@ import {
 } from './common';
 
 export namespace sys {
-  export const crypto = CryptoApi.instance();
```

Comment:
Where is the crypto instance now?  Only saw it gets removed here. How do we access the crypto functions now?

Reply:
Accessed via sys.hostApi.crypto
see https://docs.artela.network/develop/reference/aspect-lib/components/sys-hostapi#syshostapicrypto